### DISCUSSION TITLE
Regression in "start-server-and-test" HTTP NaN reported instead of 200

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3362] [Development] [answered]
- PR https://github.com/corona-warn-app/cwa-website/pull/3296 from @brianebeling updated [start-server-and-test](https://github.com/bahmutov/start-server-and-test) from 1.14.0 to 1.15.2.

- Now when Cypress tests are run, there is a message "is responding with HTTP status code NaN" instead of "is responding with HTTP status code 200".

- I reported this regression in issue https://github.com/bahmutov/start-server-and-test/issues/349.

**Comment by brianebeling** [comment]
Hey Mike, thanks for the feedback. Is that actively disrupting your development experience or can it be ignored for now?

**Reply by MikeMcC399** [reply]
@brianebeling 

> Is that actively disrupting your development experience or can it be ignored for now?

This is FYI only, otherwise I would have opened an issue. Sorry for not making that clear. It's a cosmetic problem only at this time. 🙂

**Reply by brianebeling** [reply]
Don't worry, we went ahead and made that assumption. I just wanted to clarify, so we don't act on false assumptions.

Thanks for the heads-up, I susbcribed to the issue you created and will then update our depdency if it is fixed.

**Comment by MikeMcC399** [comment, marked as answer]
- Issue https://github.com/bahmutov/start-server-and-test/issues/349 is resolved, leading to
- PR https://github.com/corona-warn-app/cwa-website/pull/3386

so this regression has been resolved.
